packages/block-logs-stream/src/fetchLogs.ts

```diff
@@ -36,29 +38,44 @@ export type FetchLogsOptions<TAbiEvents extends readonly AbiEvent[]> = {
   maxRetryCount?: number;
 };
 
-export type FetchLogsResult<TAbiEvents extends readonly AbiEvent[]> = {
+export type FetchLogsResult<abiEvents extends readonly AbiEvent[]> = {
   fromBlock: BlockNumber;
   toBlock: BlockNumber;
-  logs: GetLogsReturnType<undefined, TAbiEvents, true, BlockNumber, BlockNumber>;
+  logs: GetLogsReturnType<undefined, abiEvents, true, BlockNumber, BlockNumber>;
 };
 
 const RATE_LIMIT_ERRORS = [
+  // tests
```

Comment:
what does this comment mean?

Reply:
I use this as a mocked message in tests. I'm not sure it actually exists as an RPC response.